server/controllers/inventory/reports/changes.js

```diff
@@ -0,0 +1,130 @@
+
+/**
+ * @overview Price Report
+ *
+ * @description
+ * This file describes the price list report - it produces the list of prices to
+ * be used as a physical reference for invoicing.
+ *
+ * @requires lodash
+ * @requires ReportManager
+ * @requires inventorycore
+ */
+
+const _ = require('lodash');
+const ReportManager = require('../../../lib/ReportManager');
+const db = require('../../../lib/db');
+const core = require('../inventory/core');
+
+module.exports = inventoryChanges;
+
+const TEMPLATE = './server/controllers/inventory/reports/changes.handlebars';
+
+async function inventoryChanges(req, res, next) {
+  const params = _.clone(req.query);
+
+  const qs = _.extend(req.query, {
+    csvKey : 'groups',
+    footerRight : '[page] / [toPage]',
+    footerFontSize : '7',
+  });
+  const metadata = _.clone(req.session);
+
+  let report;
+
+  try {
+    report = new ReportManager(TEMPLATE, metadata, qs);
+    const { dateFrom, dateTo } = params;
+
+    const inventorySql = `
+      SELECT DISTINCT BUID(iv.uuid) as uuid, iv.text AS inventory_name
```

Comment:
Why do we need the current inventory items?  Can't we just use the data in the logs?